tests/system/runtimes/test_nuclio.py

```diff
@@ -85,6 +88,37 @@ def test_deploy_function_with_model_runner(self):
         resp = function.invoke("/", {"x": "y"})
         assert resp == {"x": "y", "extra": 123}
 
+    @pytest.mark.parametrize("raise_exception", [True])
```

Comment:
Seems like this was supposed to be
```suggestion
    @pytest.mark.parametrize("raise_exception", [True, False])
```